.github/pull.yml

```diff
@@ -0,0 +1,5 @@
+version: "1"
+rules:
+  - base: develop
+    upstream: astaxie:develop
```

Comment:
please change this to develop-2.0. We keep `develop` branch to fix some critical bug for v1.x.x